packages/mgt-components/src/components/mgt-person/mgt-person.ts

```diff
@@ -225,6 +225,27 @@ export class MgtPerson extends MgtTemplatedComponent {
     this.requestStateUpdate();
   }
 
+  /**
+   * usage property allows you to specify where the component is being used to add
```

Comment:
This is an interesting one. 
We need this to make a visual adjustment so that when this component is used in the context of an mgt-people component with presence enabled.
As a public property on the component, it should technically be added to the docs and need a docs PR linked to this one.

Reply:
I have updated the docs PR. What do you mean here?
> We need this to make a visual adjustment so that when this component is used in the context of an mgt-people component with presence enabled.

I figured, communication between components will work for child -> parent with some wiggling. However, parent -> child needs passing of data around to make the child aware of what the parent needs. Using this value, I can detect that. I faced a similar issue here https://github.com/microsoftgraph/microsoft-graph-toolkit/pull/1726 where I couldn't tell what child component, I'm in in the person card. Doing a CSS query on a shadow Dom is easy, but a shadow Dom inside a shadow Dom has proved a challenge.

Reply:
Yeah, I was thinking out loud, albeit incompletely, that this property only exists for parent => child communication and while we could make an argument for not documenting it would be best if we made sure that we have a linked docs PR